frontend/src/app/components/ExperimentRowCard/index.js

```diff
@@ -104,15 +104,15 @@ export class ExperimentRowCard extends React.Component {
     const { title, survey_url } = experiment;
     const surveyURL = buildSurveyURL("givefeedback", title, installed, clientUUID, survey_url);
     return (
-      <div>
+      <object>
```

Comment:
Seems to work, but what's with the `<object>` hack? What's it do, etc?

Reply:
FWIW, might help to at least have a short comment in there pointing at that blog post because it's a weird hack